front/src/Components/UI/ErrorScreen.svelte

```diff
@@ -0,0 +1,157 @@
+<script lang="ts">
+    import { fly } from "svelte/transition";
+    import { errorScreenStore } from "../../Stores/ErrorScreenStore";
+
+    import logo from "../images/logo-min-white.png";
+    import error from "../images/error.png";
+    import cup from "../images/cup.png";
+    import reload from "../images/reload.png";
+    import external from "../images/external-link.png";
+    import { get } from "svelte/store";
+
+    let errorScreen = get(errorScreenStore);
+
+    function click() {
+        if (errorScreen.urlToRedirect) window.location.replace(errorScreen.urlToRedirect);
+        else if (errorScreen.type === "redirect" && window.history.length > 2) history.back();
+        else window.location.reload();
+    }
+    let details = errorScreen.details;
+    let timeVar = errorScreen.timeToRetry ?? 0;
+    if (errorScreen.type === "retry") {
+        setInterval(() => {
+            if (timeVar <= 1000) click();
+            timeVar -= 1000;
+        }, 1000);
+    }
+
+    $: detailsStylized = details.replace("{time}", `${timeVar / 1000}`);
+</script>
+
+<main class="errorScreen" transition:fly={{ y: -200, duration: 500 }}>
+    <div style="width: 90%;">
+        <img src={logo} alt="WorkAdventure" class="logo" />
```

Comment:
The error page logo should be also be customizable.

Take a look at http://github.com/thecodingmachine/workadventure/blob/a30c2ccce7b4eaffa5e1ce511ce94d754ea7f9bd/front/src/Components/Login/LoginScene.svelte#L17-L17

You can talk to Alexis about what he did to allow customizing this part!